src/runtime/basic-runtime.js

```diff
@@ -367,7 +366,10 @@ export class ConfiguredBasicRuntime {
     }
 
     /** @private @const {!AutoPromptManager} */
-    this.autoPromptManager_ = new AutoPromptManager(this);
+    this.autoPromptManager_ = new AutoPromptManager(
+      this,
+      Promise.resolve(this.configuredClassicRuntime_)
```

Comment:
Just curious, why does this need to be a promise?

Reply:
Remove the promise. I initially used it cuz the buttonApi is using it.